src/renderer/ui/components/modals/GoToModal.js

```diff
@@ -4,6 +4,33 @@ import { findDOMNode } from 'react-dom';
 import Dialog from './ThemedDialog';
 import FlatButton from 'material-ui/FlatButton';
 import TextField from 'material-ui/TextField';
+import request from 'request';
+
+// Initialize the global Logger to forward to the main process.
+window.Logger = remote.getGlobal('Logger');
+
+const urlresolve = function urlresolve(url) {
+  return new Promise((resolve, reject) => {
+    const options = {
+      url,
+      headers: {
+        followAllRedirects: true,
+      },
+    };
+    request
```

Comment:
This sends requests from the `http` stack of node.js and some GPM URL's might require authentication to fully resolve (or they might redirect to the sign in page).  We should use `fetch` or the Electron `net` API.